packages/code-block/index.test.js

```diff
@@ -76,6 +77,7 @@ it('should have a data-heap-track="code-block-clipboard-icon" attribute on the C
   )
   const copyButton = screen.getByText('Copy')
   expect(copyButton.tagName).toBe('BUTTON')
+  expect(copyButton.getAttribute('data-heap-track')).toBe(heapAttributes.copy)
```

Comment:
This and the `heapAttributes` import are an unrelated fix for a slip I noticed when writing new tests. Previously we weren't actually testing for the `data-heap-track` attribute 😅